test/extended/builds/run_fs_verification.go

```diff
@@ -23,11 +23,11 @@ metadata:
 spec:
   runPolicy: Serial
   source:
+  	type: Dockerfile
```

Comment:
four spaces instead of two? Should align with `dockerfile` below.

Reply:
tab vs space :sweat_smile: should be fixed with latest push